test/libsolidity/syntaxTests/parsing/function_type_in_struct.sol

```diff
@@ -8,4 +8,4 @@ contract test {
 // ----
 // Warning: (49-55): Naming function type parameters is deprecated.
 // Warning: (57-63): Naming function type parameters is deprecated.
-// Warning: (83-89): Naming function type return parameters is deprecated.
+// SyntaxError: (83-89): Return parameters in function types may not be named.
```

Comment:
I think this test case should be updated to remove the name and to remove the error. The error is tested for in the other test case.